### PR TITLE
Feature/mentee crud

### PR DESCRIPTION
**Pull request by ferror18**
Read functionality plus form and buttons for create and delete

### CODE REVIEW

**Review by Nicci498 — APPROVED** [review]

File: src/components/pages/Headmaster/Mentees/MenteeProfile.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            );
+          })}
+        </>
+      ),
+    },
+  ];
+  return (
+    <div className="menteeProfileWrapper">
+      {!currentMentee ? (
+        <Skeleton />
```
Comment (Jahteo) [inline src/components/pages/Headmaster/Mentees/MenteeProfile.js:47]: I don't know what this is, but now I must go find it in and! Love the component name!

File: src/utils/debugMode.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (Jahteo) [inline src/utils/debugMode.js:1]: interesting. Is this your own creation?! I'll have to explore it later.

File: src/state/reducers/authReducer.js [hunk 3]
```diff
@@ -36,7 +37,7 @@ const authReducer = (state = initialState, action) => {
   switch (action.type) {
     case AUTH_SUCCESS:
       let decoded = parseJwt(String(action.payload));
-      // console.log('decoded.id & decoded.role: ', decoded.id, decoded.role);
+      debugLog(action.type, action.payload);
```
Comment (Jahteo) [inline src/state/reducers/authReducer.js:40]: love how simple it is to implement

**Review by Jahteo — APPROVED** [review]
Looks great as usual. I'm exited to learn about your debug logger!

**Pull request merged**